# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux26/sungem-quiet.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

This auto-neg messages always annoyed me while debugging (kernel, initrd, ...),
during general system development, or when in single-user-mode.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.25/drivers/net/sungem.c.orig	2008-04-20 11:51:29.000000000 +0200
+++ linux-2.6.25/drivers/net/sungem.c	2008-04-20 11:51:52.000000000 +0200
@@ -1433,9 +1433,9 @@
 		 */
 		if (gp->phy_mii.def->magic_aneg)
 			return 1;
-		if (netif_msg_link(gp))
+		/* if (netif_msg_link(gp))
 			printk(KERN_INFO "%s: switching to forced 100bt\n",
-				gp->dev->name);
+				gp->dev->name); */
 		/* Try forced modes. */
 		gp->phy_mii.def->ops->setup_forced(&gp->phy_mii, SPEED_100,
 			DUPLEX_HALF);
@@ -1451,9 +1451,9 @@
 			gp->phy_mii.def->ops->setup_forced(&gp->phy_mii, SPEED_10,
 				DUPLEX_HALF);
 			gp->timer_ticks = 5;
-			if (netif_msg_link(gp))
+			/*if (netif_msg_link(gp))
 				printk(KERN_INFO "%s: switching to forced 10bt\n",
-					gp->dev->name);
+					gp->dev->name); */
 			return 0;
 		} else
 			return 1;
